elvis.config

```diff
@@ -2,6 +2,7 @@
   [{config,
     [#{dirs => ["src/**"],
        filter => "*.erl",
+       ignore => ["src/rebar3_typer_prv.erl"],
```

Comment:
Lines being too long, but instead of forcing the formatter to fix those I postponed it for now

Reply:
Just split the strings into multiple ones `"like I did" "here"`… that should be enough.
But if you really really want to postpone the issue, then add an `-elvis` attribute to the module just for that rule in particular instead of ignoring all rules for the whole module.